index.bs

```diff
@@ -1,17 +1,15 @@
 <h1>Use Cases and Requirements for Element Queries</h1>
 <pre class='metadata'>
-Group: ricg
+Group: wicg
 Status: ED
-ED: http://responsiveimagescg.github.io/eq-usecases/
-Shortname: eq-usecases
+ED: https://github.com/WICG/cq-usecases
+Shortname: cq-usecases
 Level: 1
-Editor: Mat Marquis, RICG, http://responsiveimages.org
-Editor: Scott Jehl, Filament Group, http://filamentgroup.com/
-Abstract: This document captures the use cases and requirements for standardizing a solution for element queries. The <a href="#usecases">use cases</a> and <a href="#dfn-requirements">requirements</a> were gathered with consultation with the <a href="http://w3c.org/">HTML Working Group</a> and <a href="http://whatwg.org">WHATWG</a> participants, <a href="http://www.w3.org/community/respimg/"><abbr title="Responsive Images Community Group">RICG</abbr></a> group members, and the general public.
-Abstract: Found a bug or typo? Please file an <a href="https://github.com/ResponsiveImagesCG/eq-usecases/issues">issue on GitHub</a>!
-!Version History: <a href="https://github.com/ResponsiveImagesCG/eq-usecases/commits/gh-pages">Commit History</a>
-!Participate: <a href="irc://irc.w3.org:6665/#respimg">IRC: #respimg on W3C's IRC</a>
-!Participate: <a href="https://github.com/ResponsiveImagesCG/eq-usecases">GitHub</a>
+Editor: Tommy Hodgins, WICG, https://wicg.io
```

Comment:
Probably want to use https://tabatkins.github.io/bikeshed/#metadata-former-editor, as Matt and Scott did work on it originally.

Reply:
I've updated this with 'Former Editor' lines